lang/cs/Org.Apache.REEF.Client.Tests/WindowsHadoopEmulatorYarnClientTests.cs

```diff
@@ -91,19 +92,74 @@ public void TestGetClusterMetrics()
             Assert.IsTrue(clusterMetrics.ActiveNodes > 0);
         }
 
-        //// TODO: [REEF-757] Once submit API is in place, submit an app then get the details
-        ////[TestMethod]
-        ////[TestCategory("Functional")]
-        ////public void TestGetApplication()
-        ////{
-        ////    const string applicationName = @"application_1440795762187_0001";
+        [TestMethod]
+        [TestCategory("Functional")]
+        public void TestApplicationSubmissionAndQuery()
```

Comment:
Pretty sure it does. See http://stackoverflow.com/questions/2060692/how-does-one-test-async-code-using-mstest. ASA is using it as well.